File: backend-for-frontend/citizen-data-controller-backend-for-frontend/src/main/java/br/com/lucasalbertoni/master/degree/citizen/citizendatacontrollerbackendforfrontend/entity/Permissao.java
package br.com.lucasalbertoni.master.degree.citizen.citizendatacontrollerbackendforfrontend.entity;

import java.io.Serializable;

public class Permissao implements Serializable{
	
	private static final long serialVersionUID = -7806295949542870182L;
	private String cnpj;
	private TokenLiberacaoAcesso tokenLiberacaoAcesso;
	private LoginToken loginToken;
	
	public String getCnpj() {
		return cnpj;
	}
	public void setCnpj(String cnpj) {
		this.cnpj = cnpj;
	}
	public TokenLiberacaoAcesso getTokenLiberacaoAcesso() {
		return tokenLiberacaoAcesso;
	}
	public void setTokenLiberacaoAcesso(TokenLiberacaoAcesso tokenLiberacaoAcesso) {
		this.tokenLiberacaoAcesso = tokenLiberacaoAcesso;
	}
	public LoginToken getLoginToken() {
		return loginToken;
	}
	public void setLoginToken(LoginToken loginToken) {
		this.loginToken = loginToken;
	}

}

File: citizendatacontroller/lib/models/permissao.dart

import 'package:citizendatacontroller/models/tokenliberacaoacesso.dart';

import 'logintoken.dart';

class Permissao{
  final String cnpj;
  final TokenLiberacaoAcesso tokenLiberacaoAcesso;
  final LoginToken loginToken;

  Permissao(this.cnpj, this.tokenLiberacaoAcesso, this.loginToken);
}